site2/docs/deploy-bare-metal.md

```diff
@@ -19,21 +19,20 @@ sidebar_label: Bare metal
 > package and install `apache-pulsar-offloaders` under `offloaders` directory in the pulsar directory on every broker node. For more details of how to configure
 > this feature, you can refer to the [Tiered storage cookbook](cookbooks-tiered-storage.md).
 
-Deploying a Pulsar cluster involves doing the following (in order):
+Deploying a Pulsar cluster involves doing the following:
 
-* Deploy a [ZooKeeper](#deploy-a-zookeeper-cluster) cluster (optional)
-* Initialize [cluster metadata](#initialize-cluster-metadata)
-* Deploy a [BookKeeper](#deploy-a-bookkeeper-cluster) cluster
-* Deploy one or more Pulsar [brokers](#deploy-pulsar-brokers)
+1. Deploy a [ZooKeeper](#deploy-a-zookeeper-cluster) cluster (optional)  
+2. Initialize [cluster metadata](#initialize-cluster-metadata)  
+3. Deploy a [BookKeeper](#deploy-a-bookkeeper-cluster) cluster  
+4. Deploy one or more Pulsar [brokers](#deploy-pulsar-brokers)  
 
 ## Preparation
 
 ### Requirements
 
-Currently, Pulsar is available for 64-bit **macOS*, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK 8 or later versions.
+Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK 8 or later versions.
 
-> If you already have an existing ZooKeeper cluster and want to reuse it, you do not need to prepare the machines
-> for running ZooKeeper.
+> You can reuse existing Zookeeper clusters.
```

Comment:
```suggestion
> You can reuse existing ZooKeeper clusters.
```

Do we want to add > ** Tip** above this line?

Reply:
ZooKeeper